src/pycram/external_interfaces/giskard.py

```diff
@@ -794,19 +914,31 @@ def make_quaternion_stamped(quaternion: List[float]) -> 'QuaternionStamped':
     return msg
 
 
-def make_vector_stamped(vector: List[float]) -> 'Vector3Stamped':
+def make_vector_stamped(vector: List[float], frame_id: Optional[str] = "map") -> Vector3Stamped:
     """
     Creates a Vector3Stamped message, this is similar to PointStamped but represents a vector instead of a point.
 
     :param vector: The vector given as xyz in world frame
+    :param frame_id: Optional, the frame for the vector.
     :return: A Vector3Stamped message
     """
     msg = Vector3Stamped()
     msg.header.stamp = Time().now()
-    msg.header.frame_id = "map"
+    msg.header.frame_id = frame_id
 
     msg.vector.x = vector[0]
     msg.vector.y = vector[1]
     msg.vector.z = vector[2]
 
     return msg
+
+
+def _pose_to_pose_stamped(pose: PoseStamped) -> ROSPoseStamped:
```

Comment:
Put the ROSPoseStamped in qoutation marks, otherwise the import would fail in the Pip package